### DISCUSSION TITLE
Automatic currency conversion to MDL

### DISCUSSION THREAD

**Discussion by davidfilat** [alexminza/wc-moldovaagroindbank#27] [Ideas]
Hi @alexminza,
We are using the plugin on our site and it works great, however there is the small issue of currency conversion. Since we are offering additional payment gateways, the default currency on our site is EURO and we're using the [Woocommerce Booster plugin](Woocommerce Booster plugin) to convert the amounts to MDL, if the MAIB Gateway is selected at checkout. 
Now we are working on a SPA application for our store and we intend to use Woocommerce in headless mode, mainly using the [Store API](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/StoreApi/README.md). However, the conversion doesn't happen via the REST API and we have contacted the support for Woocommerce Booster plugin to see whether their plugin supports currency conversion for the REST API and they said that it cannot do that.
Is it possible for the [wc-moldovaagroindbank](https://github.com/alexminza/wc-moldovaagroindbank) plugin to handle the currency conversion automatically from EUR to MDL when the customer is redirected to the MAIB payment page?

Thank you for the great work on the plugin!

**Comment by Olya3ch** [comment]
It's a great idea and it will save us from so many troubles!

**Comment by alexminza** [comment]
> Is it possible for the [wc-moldovaagroindbank](https://github.com/alexminza/wc-moldovaagroindbank) plugin to handle the currency conversion automatically from EUR to MDL when the customer is redirected to the MAIB payment page?

Why not accept payments in multiple currencies? MAIB seems to support this and [wc-moldovaagroindbank](https://github.com/alexminza/wc-moldovaagroindbank) will just work transparently without the need to maintain a full currency conversion infrastructure which frankly is out of the scope of this project.

> = 1.1.6 =
> Added support for EUR and USD currencies

Reference: https://github.com/alexminza/wc-moldovaagroindbank/blob/master/readme.txt#L126

**Reply by davidfilat** [reply]
We've just tried it yesterday and the transactions are rejected by the bank. We'll ask them again.
